ta4j/src/main/java/org/ta4j/core/indicators/RAVIIndicator.java

```diff
@@ -44,15 +46,31 @@ public RAVIIndicator(Indicator<Decimal> price, int shortSmaTimeFrame, int longSm
         super(price);
         shortSma = new SMAIndicator(price, shortSmaTimeFrame);
         longSma = new SMAIndicator(price, longSmaTimeFrame);
+        this.toAbs = false;
+    }
+    
+    /**
+     * Constructor.
+     * @param price the price
+     * @param shortSmaTimeFrame the time frame for the short SMA (usually 7)
+     * @param longSmaTimeFrame the time frame for the long SMA (usually 65)
+     * @param toAbs if true, absolute value of indicator is returned
+     */
+    public RAVIIndicator(Indicator<Decimal> price, int shortSmaTimeFrame, int longSmaTimeFrame, boolean toAbs) {
```

Comment:
Not in the scope of this PR. If the use wants the absolute value he can use the `AbsoluteIndicator`